src/shortcuts/__tests__/pinSubthoughts.ts

```diff
@@ -0,0 +1,41 @@
+import { inputHandlers } from '../../shortcuts'
+import { importText } from '../../action-creators'
+import { RANKED_ROOT } from '../../constants'
+import { noop } from 'lodash'
+import { createTestStore } from '../../test-helpers/createTestStore'
+import { attributeEquals } from '../../selectors'
+
+it('toggle =pinChildren attribute of parent of cursor', async () => {
+
+  const store = createTestStore()
+
+  const { keyDown } = inputHandlers(store)
+
+  // import thoughts
+  await store.dispatch(importText(RANKED_ROOT, `
```

Comment:
You can actually omit `await` here. `importText` technically returns a `Promise`, but it is already resolved so `store.dispatch` returns immediately. I will change `importText` to not return a `Promise`, but Fedor is in the middle of a large refactor of that function at the moment.